### PR TITLE
Update cache_domains.json

### PR DESCRIPTION
**Pull request by nyemenzo**
### What CDN does this PR relate to
Crossfire

### Capture method
Netlimiter

### Testing Scenario
i haven't tested it yet because i dont have the knowledge to manually add a zone but i notice it in my limiter that give me this remote name.

can you teach me how to manually add zone to the container? thank you.

PS: I am from Philippines.

### CODE REVIEW

**Comment by nyemenzo** [comment]
i will edit the cdn after running wireshark. thank you. i will leave it blank first. it seems netlimiter and wireshark give diffrent host.

**Comment by VibroAxe** [comment]
Excuse my ignorance but what is crossfire?

**Comment by nyemenzo** [comment]
crossfire is a FPS game here in the Philippines. it is well known here and lots of players.

here is now their official website since they upgrade from previous one
http://forums.gameclub.ph/index.php?board=237.0

File: cache_domains.json [hunk 1]
```diff
@@ -97,6 +97,11 @@
 			"description": "CDN for xboxlive",
 			"domain_files": ["xboxlive.txt"],
 			"mixed_content": true
+		},
+		{
+			"name": "crossfire",
+			"description": "CDN for crossfire",
+			"domain_files": ["crossfire.txt"]
```
Comment (VibroAxe) [inline cache_domains.json]: This file doesn't exist in your commit?

Comment (VibroAxe) [reply cache_domains.json]: @nyemenzo ?

**Comment by VibroAxe** [comment]
Thanks for the input but I'm afraid i'm going to have to reject this due to invalid configuration, crossfire.txt in json but not included in commit. Please feel free to reopen/resubmit with these changes resolved.

**Pull request closed without merging**